deploy/gke/staging.yaml

```diff
@@ -19,5 +19,5 @@ region: us-central1
 ip: 34.107.161.252
 domain: staging.api.datacommons.org
 api_title: DataCommons API (Staging)
-nodes: 3
+nodes: 1
```

Comment:
should we keep this to at least 2, so that it's closer to prod setup (and not a lone instance?)